src/platyPS/platyPS.psm1

```diff
@@ -2782,12 +2799,28 @@ function ConvertPsObjectsToMamlModel
     $Outputs = @()
 
     $Help.returnValues.returnValue | ForEach-Object {
-        $OutputObject = New-Object -TypeName Markdown.MAML.Model.MAML.MamlInputOutput
-        $OutputObject.TypeName = $_.type.name
-        $OutputObject.Description = $_.description |
-            DescriptionToPara |
-            AddLineBreaksForParagraphs
-        $Outputs += $OutputObject
+        if ($_.description -eq $null -and $_.type.name -ne $null)
+        {
+            $Outputtypes = $_.type.name.split("`n")
```

Comment:
Continuing the conversation from #382 

You didn't specify what's the source of this INPUT.
For example take a look at
https://github.com/tlindsay42/ArmorPowerShell/pull/134/files

Original input was 

```
.INPUT
A
B
C
```

But the correct (from the powershell help system view point) way to do is

```
.INPUT
A
.INPUT
B
.INPUT
C
```

And the reason is that powershell help allows you to have a paragphas of text with a description about the .INPUT value. In this case first line `A` represents the type. And next 2 lines

```
B
C
```

represent the body.

If the problem you are addressing here is different from that, then please provide a more elaborated description of the problem with code that demonstrates it. Currently you only stated how the markdown looks like and how it should look like, but didn't show what's the PowerShell code that produced this markdown.

Reply:
Oh I see what you mean.  Our cmdlet are written in C#, so I'm not entirely sure how the input and output is generated: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/KeyVault/Commands.KeyVault/Commands/GetAzureKeyVault.cs.  If there is an easy way to tag the code so that the inputs/outputs are formatted correctly in "get-help", I'm happy to go through and do so.  At the moment though, when I compile the code and run "get-help", this is what happens:

```powershell
PS C:\Users\Maddie Clayton> $a = get-help get-azurermkeyvault -full
PS C:\Users\Maddie Clayton> $a.returnValues.returnValue

type
----
@{name=Microsoft.Azure.Commands.KeyVault.Models.PSKeyVault...

PS C:\Users\Maddie Clayton> $a.returnValues.returnValue.type

name
----
Microsoft.Azure.Commands.KeyVault.Models.PSKeyVault...

PS C:\Users\Maddie Clayton> $a.returnValues.returnValue.type.name
Microsoft.Azure.Commands.KeyVault.Models.PSKeyVault
System.Collections.Generic.List`1[[Microsoft.Azure.Commands.KeyVault.Models.PSKeyVaultIdentityItem, Microsoft.Azure.Commands.KeyVault, Version=5.0.2.0, Culture=neutral, PublicKeyToken=null]]
Microsoft.Azure.Commands.KeyVault.Models.PSDeletedKeyVault
System.Collections.Generic.List`1[[Microsoft.Azure.Commands.KeyVault.Models.PSDeletedKeyVault, Microsoft.Azure.Commands.KeyVault, Version=5.0.2.0, Culture=neutral, PublicKeyToken=null]]
```

That last output is just a string with all four output types, so when it hits the renderer, what is written is one "###" tag followed by the entire string of all four types.  If there is an easy way to fix this in the C# code, please let me know.  I made the changes to the renderer so that all four types would get their own "###" tag, but I see how that could possibly be an issue - though shouldn't the description be put into the "description" field? I am currently filtering to ensure that anything with a "description" field is rendered in the old way.

Reply:
Unfortunately still has the same issue :(.  Okay, I'll modify if to be for output only, and push another commit.

Reply:
Actually seeing the exact same issue with inputs as outputs

Reply:
Can you link the code that produces the input?

Reply:
Ah I see, all `ValueFromPipeline` and `ValueFromPipelineByPropertyName` are turned into input. Ok, I can see the same behavior on a simple example:

```
function foo { [OutputType([int])] param([Parameter(ValueFromPipeline)][string]$s, [Parameter(ValueFromPipelineByPropertyName)][int]$k) }
```

Alright, we shell keep both then!